services/github/src/handlers.js

```diff
@@ -406,14 +409,18 @@ async function statusHandler(message) {
         check_run_id: checkRun.checkRunId,
       });
     } else {
+      const taskDefinition = await this.queueClient.task(taskId).catch(debug);
+      debug(`Result status. Got task build from DB and task definition for ${taskId} from Queue service`);
+
       const checkRun = await instGithub.checks.create({
         owner: organization,
         repo: repository,
-        name: `Task ${taskId}`,
+        name: `${taskDefinition.metadata.name}: task ${taskId}`,
         head_sha: sha,
-        output: { // TODO: a helpful output
-          title: `TaskGroup: queued for ${eventType})`,
-          summary: `Check for ${eventType}`,
+        output: {
```

Comment:
I find myself wondering if we should not report group status. Is the purpose of it to allow somebody to block on us being entirely done?

Reply:
What do you mean? (I'm not reporting group status, just add the link to the task group in the task. I find myself wanting to see the whole taskgroup often, but now there's no link to it)